src/download/download-version.ts

```diff
@@ -108,12 +108,9 @@ async function downloadVersion(
   await validateChecksum(checkSum, downloadPath, arch, platform, version);
 
   let uvDir: string;
-  const extension = getExtension(platform);
   if (platform === "pc-windows-msvc") {
-    const fullPathWithExtension = `${downloadPath}${extension}`;
-    await fs.copyFile(downloadPath, fullPathWithExtension);
-    uvDir = await tc.extractZip(fullPathWithExtension);
     // On windows extracting the zip does not create an intermediate directory
+    uvDir = await tc.extractTar(downloadPath, undefined, "x");
```

Comment:
What is "x" ?

Reply:
It means "extract", default is "xz" which means extract and gzip. Testing shows that works too, but it's technically not a gzip file so I wanted to be explicit.